draft-ietf-tls-tls13.md

```diff
@@ -1583,16 +1584,19 @@ legacy_compression_methods
 extensions
 : Clients request extended functionality from servers by sending
   data in the extensions field.  The actual "Extension" format is
-  defined in {{extensions}}.
+  defined in {{extensions}}.  In TLS 1.3, support for and the presence
```

Comment:
The grammar here is unnecessarily complex, and could be confusing. I suggest simply "In TLS 1.3, use of certain extensions is mandatory, ...".